src/main/scala/akka/contrib/process/NonBlockingProcess.scala

```diff
@@ -181,6 +181,10 @@ class NonBlockingProcess(
         val stdin =
           Sink
             .foreach[ByteString](bytes => nuProcess.writeStdin(bytes.toByteBuffer))
+            .mapMaterializedValue { future =>
+              future.onComplete(_ => nuProcess.closeStdin(false))
```

Comment:
Can you name the boolean arg so it's more obvious -- i.e. `closeStdin(force = false)`?

Reply:
That's actually not possible because we're calling into compiled java code which doesn't track argument names

Reply:
Good point. :-)